### PR TITLE
Add auto download feature

### PR DESCRIPTION
**Pull request by muratugureminoglu**

### CODE REVIEW

File: docker/Dockerfile_Process [hunk 1]
```diff
@@ -20,7 +21,13 @@ RUN cd home \
 
 RUN cd home \
     && pwd \
-    && ./install_ant-media-server.sh -i ${AntMediaServer} -s false
+    && if [ -n "$AntMediaServer" ]; then \
+           ./install_ant-media-server.sh -i ${AntMediaServer} -s false; \
+       fi \
+    && if [ -n "$LicenseKey" ]; then \
```
Comment (mekya) [inline docker/Dockerfile_Process:27]: I think it will execute both if statements even if first is successful.

File: docker/Dockerfile_Process [hunk 2]
```diff
@@ -20,7 +21,13 @@ RUN cd home \
...
+           ./install_ant-media-server.sh -l ${LicenseKey} -s false; \
+       fi
+
```
Comment (mekya) [inline docker/Dockerfile_Process:30]: What if there is no {AntMediaServer} and {LicenseKey}?

**Review by mekya — CHANGES_REQUESTED** [review]
Could you please check comments?

File: docker/Dockerfile_Process (hunk 1 above)
Comment (muratugureminoglu) [reply docker/Dockerfile_Process:27]: Yes, you are right. Let's put in if statements.

File: docker/Dockerfile_Process (hunk 2 above)
Comment (muratugureminoglu) [reply docker/Dockerfile_Process:30]: Let's do that by adding an exit code.

**Review by mekya — APPROVED** [review]

**Pull request merged**